app/views/movies/_modal_content.html.erb

```diff
@@ -9,8 +9,7 @@
     </div>
     <div class="watch-info">
       <%= render 'movies/screenings', movie: movie %>
-      <%= render "movies/mini_modal/streaming_service_providers",
-          streaming_service_providers: movie.streaming_service_providers %>
```

Comment:
I've condensed display logic across the app so we don't need 2 different (but very similar) partials anymore